saleor/order/models.py

```diff
@@ -82,7 +82,18 @@ def ready_to_confirm(self):
         return self.filter(status=OrderStatus.UNCONFIRMED)
 
 
+def get_order_number():
+    with connection.cursor() as cursor:
+        cursor.execute("SELECT nextval('order_order_number_seq')")
+        result = cursor.fetchone()
+        return result[0]
```

Comment:
Isn't there a chance that this will cause races? I would say we should have postgres do auto-increment rather than making a SQL query to get the next value